lib/common_ui/primary_container.dart

```diff
@@ -1,14 +1,21 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
-import 'package:scouts_system/view_model/students.dart';
 
 // ignore: must_be_immutable
```

Comment:
Why are variables not final? It is a stateless widget. You rebuild the widget if needed and do not update its state, right?

Reply:
solved #14